### PR TITLE
Add replication enabled attribute in pi volume resource

### PR DESCRIPTION
**Pull request by dhirendersingh19**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

=== RUN   TestAccIBMPIVolumebasic
--- PASS: TestAccIBMPIVolumebasic (130.20s)
PASS
=== RUN   TestAccIBMPIVolumePool
--- PASS: TestAccIBMPIVolumePool (94.04s)
PASS
=== RUN   TestAccIBMPIVolumeGRS
--- PASS: TestAccIBMPIVolumeGRS (188.75s)
PASS
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_volume_test.go [hunk 1]
```diff
@@ -160,3 +160,64 @@ func testAccCheckIBMPIVolumePoolConfig(name string) string {
...
+	resource "ibm_pi_volume" "power_volume"{
+		pi_volume_size       = 20
+		pi_volume_name       = "%[1]s"
+		pi_volume_pool       = "%[3]s"
+		pi_volume_shareable  = true
+		pi_cloud_instance_id = "%[2]s"
+		pi_replication_enabled = false
+	  }
+	`, name, acc.Pi_cloud_instance_id, acc.PiStoragePool)
+}
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_volume_test.go:223]: These 2 funcs can be combined with`replication` as a parameter.

File: ibm/service/power/resource_ibm_pi_volume_test.go [hunk 2]
```diff
@@ -160,3 +160,64 @@ func testAccCheckIBMPIVolumePoolConfig(name string) string {
 	  }
 	`, name, acc.Pi_cloud_instance_id)
 }
+
+func TestAccIBMPIVolumeGRS(t *testing.T) {
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_volume_test.go:164]: What is GRS?
A func comment would like understand the test functionality.

**Review by yussufsh — CHANGES_REQUESTED** [review]
minor changes needed

File: ibm/service/power/resource_ibm_pi_volume_test.go (hunk 2 above)
Comment (dhirendersingh19) [reply ibm/service/power/resource_ibm_pi_volume_test.go:164]: volume replication comes under global replication service(GRS)

File: ibm/service/power/resource_ibm_pi_volume_test.go (hunk 1 above)
Comment (dhirendersingh19) [reply ibm/service/power/resource_ibm_pi_volume_test.go]: test case config refactored

**Review by hkantare — APPROVED** [review]

**Pull request merged**